packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -3,20 +3,22 @@ import type { WaferMap } from '.';
 
 export const template = html<WaferMap>`
     <div class="wafer-map-container">
-        <svg class="svg-root ${x => x.orientation}">
-            <g class="zoom-container">
-                <svg
-                    class="circle-base"
-                    version="1.1"
-                    x="0px"
-                    y="0px"
-                    viewBox="1 .45 20 21"
-                >
-                    <path
-                        class="circle-drawing-path"
-                        d="m 21 12 a 10 10 330 1 1 0 -1.98 a 1 1 0 0 0 0 2"
-                    />
-                </svg>
+        <svg class="svg-root">
+            <g class="zoom-container" ${ref('zoomContainer')}>
+                <g class="${x => x.orientation}">
```

Comment:
based on the above suggestion we would add a class like `notch`

```suggestion
                <g class="notch ${x => x.orientation}">
```